### ISSUE TITLE
[Feature Request] Notifications Badge

### ISSUE THREAD

**Issue by LeonardSSH** [adisreyaj/flare#24]
A small badge next to the notification icon indicating unread notifications.

![image](https://user-images.githubusercontent.com/35312043/155893980-8ba95fce-3832-402f-ac3c-d9902ac2f1f6.png)

**Comment by mamane19** [comment]
Hey @LeonardSSH,
A more permanent solution for notifications would be using Nest Events or Socket.io.

What do you think!?

**Comment by adisreyaj** [comment]
Yes. Sockets are the way to go, but introduces additional complexity at the moment. But definitely the way to go.

**Comment by mamane19** [comment]
Please when you'd like to work on it you can ping, I would love to contribute!

**Comment by adisreyaj** [comment]
Awesome. Will let you know.
